Sources_v3/StreamChatUI/Utils/UIColor+Extensions.swift

```diff
@@ -38,4 +38,6 @@ extension UIColor {
     static let incomingMessageBubbleBorder = UIColor.black.withAlphaComponent(0.08)
     static let chatBackground = UIColor(rgb: 0xfcfcfc)
     static let messageTimestamp = UIColor.lightGray
+    static let squareButtonTint = UIColor(rgb: 0x7a7a7a)
+    static let sendButtonTint = UIColor.black.withAlphaComponent(0.8)
```

Comment:
Why is this called "tint" when it's a color? :) Why is this color not specified in `UIConfig`?